eventhub/Module.md

```diff
@@ -58,33 +25,44 @@ event hubs. It includes operations to revoke publisher, get revoked publishers,
 
 * An Event Hub https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create#create-an-event-hub 
 
+### Obtaining Tokens
 * Connection String of the Event Hub Namespace
-We need management credentials to communicate with the Event Hubs. These credentials are available in the connection 
-string of the Event Hub namespace. Obtain the connection string for the Event Hubs namespace by following the 
-instructions given below.
-https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string#get-connection-string-from-the-portal
+
+    We need management credentials to communicate with the Event Hubs. These credentials are available in the connection 
+    string of the Event Hub namespace. Obtain the connection string for the Event Hubs namespace by following the 
+    instructions given below.
+    https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string#get-connection-string-from-the-portal
+    (Eg: "Endpoint=sb://<Namespace_Name>.servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=i1f2BXAbmtxhu............f9GMyfnNWvQ3CY=")
 
 * Shared Access Signature (SAS) Authentication Credentials
-You need to extract the Shared Access Key Name, Shared Access Key, Resource URI to the Event Hub Namespace separately 
-from the connection string.
-    * Shared Access Key Name, 
-    * Shared Access Key, 
-    * Resource URI to the Event Hub Namespace.
 
-# Quickstart(s):
+    You need to extract the Shared Access Key Name, Shared Access Key, Resource URI to the Event Hub Namespace separately 
```

Comment:
1. Extract the Shared Access Key Name, Shared Access Key, and Resource URI from the connection string.
Let's add a sample connection string and help them identify these values.

Reply:
Is the capitalization correct? I think we can use shared access key name, shared access key, and resource URI

Reply:
Fixed in df7c75b . Thank you.